_layouts/base.html

```diff
@@ -42,7 +42,7 @@ <h4 style="padding: 0.5rem 1rem; margin-bottom: 0">Canonical Webteam practices</
           <ul class="p-navigation__links">
             <li class="p-navigation__link"><a href="/practices/">Home</a></li>
             <li class="p-navigation__link"><a href="https://github.com/canonical-webteam/practices" class="p-link--external">GitHub repo</a></li>
-            <li class="p-navigation__link"><a href="https://github.com/canonical-webteam/practices/blob/master/CONTRIBUTING.md" class="p-link--external"><code>CONTRIBUTING.md</code></a></li>
+            <li class="p-navigation__link"><a href="https://github.com/canonical-webteam/practices/blob/master/CONTRIBUTING.md" class="p-link--external">Contribute</a></li>
```

Comment:
Could we change the text to "How to contribute"? I would expect a "Contribute" link to take me to the repo itself, or a page where I can create a PR or something.

Reply:
Or maybe "Contributing guidelines" or something. Up to you.

Reply:
Sure, I wonder how necessary this link is anyway (next to GitHub link). As it is targeted to developers GitHub is where they will look for means to contribute.